src/ckb/schema/pcn.mol

```diff
@@ -1,6 +1,7 @@
 import blockchain;
 
-array Signature [byte; 65];
+array Signature [byte; 64];
+array Byte66 [byte; 66];nonce
```

Comment:
```suggestion
array Byte66 [byte; 66]; // nonce
```

Reply:
Byte66 is changed to PubNonce. I currently serialize Signature to the compact form. So it is 64 bytes. I don't fully understand the implication of using 65 bytes form of signature vs 64 bytes of signature. Can you elaborate on that?